src/shared/machine-credential.c

```diff
@@ -30,13 +30,11 @@ void machine_credential_free_all(MachineCredential *creds, size_t n) {
 
 int machine_credential_set(MachineCredential **credentials, size_t *n_credentials, const char *cred_string) {
```

Comment:
We usually introduce `MachineCredentials` or `-Context` for this. But can be done later.